webhook/app.yaml

```diff
@@ -1,3 +1,3 @@
 runtime: python38
 service: webhook
-
+entrypoint: uvicorn --port $PORT
```

Comment:
missing `host` [option](https://www.uvicorn.org/settings/). The default one is `127.0.0.1`, which will make it impossible to receive requests when it's deployed in production.

Reply:
Done